### ISSUE TITLE
Accessibility audit

### ISSUE THREAD

**Issue by smythp** [sigstore/docs#245]
## Background

While the Doks theme for Hugo is supposed to have a strong lighthouse score and seems to do some of the basics well from an accessibility perspective, a full audit with both automated and manual testing should be done now that we're on a new platform.

## Work Plan

1. Perform automated tests using axe and lighthouse.
2. Test manually and create a report for WCAG compliance, at least to A but ideally higher.
3.  Create issues for existing accessibility considerations.

Once a report has been created and issues laid out, I'll close this issue.

## Notes

There is a junior a11y consultant I'm mentoring and I'd like to bring him in for some of the initial work. He is exclusively a screen reader user. I'll discuss with @ltagliaferri.

**Comment by krishnabodawala** [comment]
Hi, my name is Krishna Bodawala. I'll be working with @smythp and Iota as a screen reader tester on this accessibility audit.
